docs/content/querying/groupbyquery.md

```diff
@@ -182,6 +182,7 @@ When using the "v2" strategy, the following runtime properties apply:
 |`druid.query.groupBy.bufferGrouperMaxLoadFactor`|Maximum load factor of the off-heap hash table used for grouping results. When the load factor exceeds this size, the table will be grown or spilled to disk. Set to 0 to use a reasonable default.|0|
 |`druid.query.groupBy.maxMergingDictionarySize`|Maximum amount of heap space (approximately) to use for the string dictionary during merging. When the dictionary exceeds this size, a spill to disk will be triggered.|100000000|
 |`druid.query.groupBy.maxOnDiskStorage`|Maximum amount of disk space to use, per-query, for spilling result sets to disk when either the merging buffer or the dictionary fills up. Queries that exceed this limit will fail. Set to zero to disable disk spilling.|0 (disabled)|
+|`druid.query.groupBy.tmpDir`|Path where temporary files created while processing the query should be stored. If specified, this configuration takes priority over the default `java.io.tmpdir` path.|
```

Comment:
How about putting this in DruidProcessingConfig? It feels more like something that should apply to any query that needs a temp scratch area. Kind of senseless for each query to manage its own scratch areas.